README.md

```diff
@@ -20,22 +20,40 @@ And we just looooove lots of lints. :blue_heart:
 ---
 
 ## Usage
-Firstly add a dev dependency in your `pubspec.yaml`:
+#### Installation
+For a start please make sure you work with latest version of Flutter & Dart.
+```yaml
+environment:
+  sdk: ">=2.18.6 <3.0.0"
+  flutter: ^3.3.10
```

Comment:
I deleted it because for some reasons most of the libraries don't write it 🤔 
Perhaps, it's because to get highest pub scoring, the library must depend on latest dependencies

Reply:
isn't those a latest dependencies?

Reply:
![image](https://user-images.githubusercontent.com/17902851/211321088-0bd2fa5b-d77e-4770-8793-1a73a9f57688.png)

Reply:
![image](https://user-images.githubusercontent.com/17902851/211321242-ed97ca66-1093-4ffe-9fa5-a38d807b8325.png)